api/src/queries.js

```diff
@@ -0,0 +1,24 @@
+const items = {
+  get: `{
```

Comment:
Did some further testing and just realised we also need to declare our vars in the query! 
```suggestion
  get: `query vars($code: string){
```